jumpcloud-ADMU/Powershell/InvokePester.ps1

```diff
@@ -2,7 +2,14 @@
 param (
     [Parameter()]
     [System.string]
-    $ModuleVersionType
```

Comment:
Several changes to how we are invoking these tests. 

Tests can be invoked by tags (much like the powershell module). In CI the tags are split into three different test runner jobs, one of the runners will always run the jobs that require the Agent be installed, the other runners will attempt to split the tags into two different groups.